.gitlab/one-pipeline.locked.yaml

```diff
@@ -0,0 +1,2 @@
+include:
+  - remote: https://gitlab-templates.ddbuild.io/libdatadog/one-pipeline/ca/05e116339b9780a138a474d0348e97debfca97f27bbc4ca489cf4e4c90d9cc94/one-pipeline.yml
```

Comment:
Where does `05e116339b9780a138a474d0348e97debfca97f27bbc4ca489cf4e4c90d9cc94` come from? 🤔 How do we know how and when to update this value?

Reply:
This is something I need to document. Its a sha of all the files that are exported as part of one-pipeline.

The tooling will automatically detect when this needs to be updated in your Repository (e.g. only when the files change).